src/components/screens/IntegrationsCatalog/RecipesDetailScreen/RecipesDetailScreen.js

```diff
@@ -257,6 +242,9 @@ export const RecipesDetailScreen = ({ path, location, pageContext }) => {
         <RecipeItemDetail {...recipe} />
         <IntegrationsAsideContainer>
           <ReadMe>
+            <span hidden id="recipe-index-title">
+              {`Recipes » How to setup ${displayName}`}
```

Comment:
Recipes have both a topic, e.g. Tailwind, and a target, e.g. React (in Storybook). Does `displayName` encompass both?

Reply:
Not yet. We haven't started adding recipes for a single package for different frameworks yet